src/assets/_project/_blocks/components/link/_qg-link.scss

```diff
@@ -19,6 +19,7 @@
     // &.hover {
     //   background: none;
     // }
+    display: inline-block;
```

Comment:
This should not be applied globally. It has side effects on line spacing and wrapping.

Reply:
I think increasing line heights is inevitable to fix this issue, because the style lacks some vertical spacing to fit in both outline and underline. I excluded digital dashboard from the change.